src/Service/MagicLinks/MagicLinkService.cs

```diff
@@ -64,6 +63,10 @@ private async Task EnforceQuotaAsync(MagicLinkTokenRequest request)
         }
     }
 
+    private static bool IsAdminConsole(PerTenant account) =>
+        string.Equals(account.Tenant, "admin", StringComparison.OrdinalIgnoreCase)
+        || string.Equals(account.Tenant, "adminconsole", StringComparison.OrdinalIgnoreCase);
```

Comment:
We should probably do something to prevent regular users from creating apps with these names? For example if `adminconsole` already exists and the user creates `admin`.